package-configuration/nginx/conf/default.conf.template

```diff
@@ -116,6 +109,18 @@ server {
 }
 
 server {
+
+  listen              *:{{ config_templates.__params_nginx.rest_internal_port }};
```

Comment:
please add this also to https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/nginx/default.conf